cmd/helm/root.go

```diff
@@ -74,6 +74,7 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 		Use:                    "helm",
 		Short:                  "The Helm package manager for Kubernetes.",
 		Long:                   globalUsage,
+		Run:                    runHelp,
```

Comment:
Could you in-line this as you did for the `options` command?
Then can you put a comment to why we need to specify the `Run` field?
I'm worried that someone could remove it during a refactor and wouldn't notice the impact.